app/services/dropdown_picker_service.js

```diff
@@ -0,0 +1,38 @@
+const getPickerFormat = (items) => {
+  let data = [];
+  items.map((item) => {
+    let itemObj = {value: item};
```

Comment:
@limkimsan consider building a language dictionary in e.g: 

lang_dict = { "km": "ខ្មែរ", "en": "English", ..etc.};
lang = {label: lang_dict[item], value: item};
languages.push(lang)